src/main/controllers/features/sole-trader/sectorYouWorkInController.ts

```diff
@@ -76,16 +76,15 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
             if (req.body.sectorYouWorkIn === "OTHER") {
                 res.redirect(addLangToUrl(BASE_URL + SOLE_TRADER_WHICH_SECTOR_OTHER, lang));
             } else {
-                if (acspData) {
-                    acspData.workSector = req.body.sectorYouWorkIn;
-                }
                 //  save data to mongodb
+                acspData.workSector = req.body.sectorYouWorkIn;
```

Comment:
So does this mean acspData can never be null?

Reply:
technically yes but putting back how it was as that is more secure and better option.